src/history/HistoryTestsUtils.cpp

```diff
@@ -665,13 +665,10 @@ CatchupSimulation::getCatchupMetrics(Application::pointer app)
 
     auto bucketsApplied = bucketApplySuccess.count();
 
-    auto& downloadTransactionsCached = app->getMetrics().NewMeter(
```

Comment:
`downloadLedgersCached` on line 642 should be removed as well, as it's not a valid metric anymore.